hathor/p2p/sync_v2/agent.py

```diff
@@ -721,7 +721,7 @@ def handle_blocks_end(self, payload: str) -> None:
         assert self.protocol.connections is not None
 
         if self.state is not PeerState.SYNCING_BLOCKS:
-            self.log.error('unexpected BLOCKS-END', state=self.state)
+            self.log.error('unexpected BLOCKS-END', state=self.state, response_code=response_code)
```

Comment:
Wouldn't it be better to use `response_code.name` instead? Since just `response_code` is considered an `int` and will end up as such in the json logs. We've already passed it through the enum so it is guaranteed to have a name.

Reply:
Nice catch. Thanks!